README.md

```diff
@@ -1,6 +1,6 @@
 # Redmine-Tracky
 
-Improved time tracking for Redmine
+Improved time tracking for Redmine (tested using Redmine 5.1)
```

Comment:
This is a pity, since it was developed for Redmine 4.x. Would be nice if we could run the test suite against multiple versions.

Reply:
yes, thought the same 👍🏼  I created [an issue](https://redmine.renuo.ch/issues/21795) and will take care of it